metadata_layer.py

```diff
@@ -471,32 +869,25 @@ def __init__(
             contributors=None,
             measurements=None,
             links=None,
-            formats=None,
-            rights_uri=None,
-            last_update_time=None,
-            circulation=None,
+            provider_entry_updated=None,
```

Comment:
in this case we would consider the metadata wrangler a remote data source as well. we manage it, but it's remote to the circulation manager. it would be from the last time we checked the metadata wrangler, not the last time the metadata wrangler checked its remote sources.